### ISSUE TITLE
How to delete an application?

### ISSUE THREAD

**Issue by RichardJECooke** [bottlesdevs/programs#138]
I tried installing Battle.net in a new Bottle. It installed but didn't work. Reported a bug on Wine Bugzilla.

I tried uninstalling it through the hamburger menu in Bottles. Nothing happens. It stays in the list.

I tried running the installer again. A new program appears called `Battle` that seems to be the same thing as `Battle.net`.

I deleted the Bottle, made a new one. But 

1) Where are the apps stored on disk so I can delete them manually instead of deleting the entire Bottle?
2) Does deleting the Bottle actually delete the app?
3) This is a bug, obvz.

**Comment by Ethanol6** [comment]
Yeah, same issue here. Depending if you are using the Flatpak version or not the files are located in different places. This part of the documentation gives you the file paths. https://docs.usebottles.com/flatpak/migrate-bottles-to-flatpak
